stdlib/format.ml

```diff
@@ -395,18 +399,19 @@ let format_pp_token state size = function
 
   | Pp_break { fits; breaks } ->
     let before, off, _ = breaks in
+    let before_width = pp_string_width state before in
```

Comment:
If this is potentially slightly expensive, maybe move it in the `Some` case below where we need the result? (It does not sound completely bizarre to call, say, `Format.pp_print_text` at the toplevel.)